CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 FEATURES:
 
   * **New resource: `digitalocean_floating_ip`** [GH-3748]
+  * **New provider: `cloudinit` - Multipart cloudinit [GH-4061], [GH-3416]
```

Comment:
There is no need to update the CHANGELOG on pull requests. We tend to find this results in merge conflicts, so upon merging we update it directly on the `master` branch.